Description: Add patch to acpi plugin to call acpi without -B as requiered by
 acpi in lenny and newer.  Upstream has rewritten plugins which fixes this
 issue.
Origin: other, http://bugs.debian.org/523239
Bug-Debian: http://bugs.debian.org/523239
Index: munin-1.2.6/node/node.d/acpi.in
===================================================================
--- munin-1.2.6.orig/node/node.d/acpi.in	2009-05-29 13:02:40.000000000 +0200
+++ munin-1.2.6/node/node.d/acpi.in	2009-05-29 13:02:57.000000000 +0200
@@ -32,9 +32,9 @@
 #%# capabilities=autoconf
 
 if [ "$1" = "autoconf" ]; then
-	acpi_available 2>/dev/null >/dev/null || ${acpi:-acpi} -t -B 2>/dev/null >/dev/null
+	acpi_available 2>/dev/null >/dev/null || ${acpi:-acpi} -t 2>/dev/null >/dev/null
         if [ "$?" = "0" ]; then
-		acpi -t -B 2>/dev/null | grep '\d' >/dev/null 2>/dev/null
+		acpi -t 2>/dev/null | grep '\d' >/dev/null 2>/dev/null
 		if [ "$?" = "0" ]; then
 		    echo yes
 		    exit 0
@@ -59,4 +59,4 @@
         exit 0
 fi
 
-${acpi:-acpi} -t -B | awk '{print "cpu.value " $4}'
+${acpi:-acpi} -t | awk '{print "cpu.value " $4}'
